code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -27,8 +27,7 @@
 	butcherable = 1
 
 	var/absorb_rate = 2 // how much item health is removed per tick when absorbing
-	var/absorb_per_health = 3 // how much resources we get per item health
-	var/absorb_completion = 6 // how much resources we get after the item is totally eaten
+	var/absorb_per_health = 1 // how much resources we get per item health
```

Comment:
Perhaps a more sane name like `resources_per_health` since it can be confusing when compared with `absorb_rate` and `absorb` later on